Source/WebKit/UIProcess/ios/WebPageProxyIOS.mm

```diff
@@ -1555,7 +1555,13 @@ static RecommendDesktopClassBrowsingForRequest desktopClassBrowsingRecommendedFo
         auto applicationName = policies.applicationNameForDesktopUserAgent();
         if (applicationName.isEmpty())
             applicationName = applicationNameForDesktopUserAgent();
-        policies.setCustomUserAgent(standardUserAgentWithApplicationName(applicationName, emptyString(), UserAgentType::Desktop));
+        // FIXME: This is done here for adding a UA override to tiktok.
+        // needsCustomUserAgentOverride() is currently very generic on purpose.
+        // In the future we want to pass more parameters for targeting specific domains.
+        if (Quirks::needsCustomUserAgentOverride(request.url()))
```

Comment:
Oh no! We need to do the Quirks check again here. How do we disable this quirk otherwise?

Reply:
rha. sorry about that. need to fix that. I will follow up with  https://bugs.webkit.org/show_bug.cgi?id=290953